src/strings/PowerFxResources.en-US.resx

```diff
@@ -5306,4 +5306,8 @@
     <value>Use the And operator to combine multiple predicates into the second argument.</value>
     <comment>{Locked=And} Limiting the Filter funciton to just two arguments, like the LookUp function, allowing us to add a third argument later also like LookUp.</comment>
   </data>
+  <data name="ErrSetVariableWithRelationshipNotAllow" xml:space="preserve">
+    <value>Can't assign variable from expandable types. For instance, a record from a datasource containing relationships.</value>
+    <comment>This will prevent traversing relationships, which can cause performance problems.</comment>
```

Comment:
>comment [](http://example.com/codeflow?start=5&length=7)

I think <comment> is meant as a hint to localization team; not shown to end users.

Reply:
Yes. I was just not sure what to include here.